### DISCUSSION TITLE
GDK_SCALE and GDK_DPI_SCALE don't scale UI anymore

### DISCUSSION THREAD

**Discussion by hpetrus** [audacity/audacity#4335] [Q&A] [answered]
GDK_SCALE and GDK_DPI_SCALE don't scale UI anymore on Arch Linux.  The version of Audacity in question is 3.2.0 alpha.

You can try `export GDK_SCALE=2` and `export GDK_DPI_SCALE=0.5` and run `audacity`, as per usual to get an unpredictable UI scale-up, but it will no longer do anything.  Everything is tiny on a HiDPI display.  Way too hard on the eyes.

Is there a new solution?

**Comment by lb90** [comment, marked as answer]
Hello! The GDK/Wayland backend doesn't read the `GDK_SCALE` environment variable at all, only the GDK/X11 backend does. You may try launching audacity with the X11 backend from a terminal:

`GDK_BACKEND=x11 audacity`

Hope it helps!

**Reply by xaionaro** [reply]
Did not help me:
```
GDK_DPI_SCALE=0.5 GDK_SCALE=2 GDK_BACKEND=x11 /usr/bin/audacity
```

No scaling. The same if I change to 0.25 and 4, if I remove the DPI variable.

The environment is Hyprland (Wayland).
